### PR TITLE
Add medication request reference

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: src/medication-dispense/medication-dispense.resource.tsx [hunk 1]
```diff
@@ -39,16 +39,20 @@ export function initiateMedicationDispenseBody(
...
       status: "on-hold", // might need to change this to appropriate status
+      authorizingPrescription: {
+        reference: "MedicationRequest/" + medicationRequest.id,
+        type: "MedicationRequest",
+      },
       medicationReference: medicationRequest.medicationReference,
       medicationCodeableConcept: medicationRequest.medicationCodeableConcept,
       subject: medicationRequest.subject,
-      performer: [
+      /* performer: [
```
Comment (mogoodrich) [inline src/medication-dispense/medication-dispense.resource.tsx:49]: I just commented this out for now, so we don't forget... the person who dispenses the medication should not be defaulted to the person who prescribed the medication... we can figure out the appropriate person to include here later.

File: src/medication-dispense/medication-dispense.resource.tsx [hunk 2]
```diff
@@ -39,16 +39,20 @@ export function initiateMedicationDispenseBody(
     let dispense = {
       resourceType: "MedicationDispense",
       status: "on-hold", // might need to change this to appropriate status
+      authorizingPrescription: {
+        reference: "MedicationRequest/" + medicationRequest.id,
+        type: "MedicationRequest",
+      },
```
Comment (mogoodrich) [inline src/medication-dispense/medication-dispense.resource.tsx:45]: @pirupius  @ibacher  just interested if manually building the reference this way is "correct"... in this case, we have pulled down the entire medication request, not the the reference to it, so we didn't seem to have the reference version available.

Comment (ibacher) [reply src/medication-dispense/medication-dispense.resource.tsx:45]: Yeah, I think it's fine to do. You don't necessarily need the `type`, though it's probably harmless.

**Review by ibacher — APPROVED** [review]

**Comment by mogoodrich** [comment]
Thanks @ibacher !  @pirupius  are you good with merging this one in?

**Review by pirupius — APPROVED** [review]
LGTM. Was waiting on Ian's feedback as the FHIR god 😄

**Comment by mogoodrich** [comment]
No worries on waiting on @ibacher , thanks @pirupius !

**Pull request merged**